src/tk/wurst_client/mods/BaseFinderMod.java

```diff
@@ -60,94 +69,51 @@ public void onUpdate()
 		if(hasTimePassedM(3000))
 		{
 			matchingBlocks.clear();
+			
+			int playerX = (int) Math.floor(mc.thePlayer.posX);
+			int playerY = (int) Math.floor(mc.thePlayer.posY);
+			int playerZ = (int) Math.floor(mc.thePlayer.posZ);
+
 			for(int y = range; y >= -range; y--)
-			{
 				for(int x = range; x >= -range; x--)
-				{
 					for(int z = range; z >= -range; z--)
 					{
-						int posX = (int)(mc.thePlayer.posX + x);
-						int posY = (int)(mc.thePlayer.posY + y);
-						int posZ = (int)(mc.thePlayer.posZ + z);
+						int posX = playerX + x;
```

Comment:
@CisBetter
In C/C++ there is the problem of allocate and deallocate memory all the time, I programmed in C a lot this year and I know why you thought that it would be a performance issue in this for loop.

But (greatly) in Java is different! Like @yellowlight2 said, it doesn't deallocate in a local context. 

Another thing is that each time you do "int x;" x is assign to 0, like implicitly "int x = 0;" so neither undeclared variables would be an issue, it is just a matter of personal coding hah.